templates/recovery.txt

```diff
@@ -4,7 +4,7 @@
 {{t "Remember password? Sign in."}} {{{ signInUrl }}}
 {{t "Reset password"}} {{{link}}}
```

Comment:
Can you add a space above this line? With the long links this looks squashed too:

```
Remember password? Sign in. http://127.0.0.1:3030/signin?email=testuser%40testuser.com&utm_source=email&utm_medium=email&utm_campaign=fx-forgot-password&utm_content=fx-remember-password
Reset password http://127.0.0.1:3030/v1/complete_reset_password?token=47b22cd271963448cf36da95cccfcfb342b5693d66f58aa635f9a95579431002&code=ae35999f861ffc81d594034eb4560af8&email=testuser%40testuser.com&service=sync&redirectTo=https%3A%2F%2Fredirect.com%2F&resume=eyJjYW1wYWlnbiI6bnVsbCwiZW50cnlwb2ludCI6bnVsbCwiZmxvd0lkIjoiM2Q1ODZiNzY4Mzc2NGJhOWFiNzhkMzMxMTdjZDU4Y2RmYjk3Mzk5MWU5NTk0NjgxODBlMDUyMmY2MThhNmEyMSIsInJlc2V0UGFzc3dvcmRDb25maXJtIjp0cnVlLCJ1bmlxdWVVc2VySWQiOiI1ODNkOGFlYS00NzU3LTRiZTQtYWJlNC0wZWQ2NWZhY2Y2YWQiLCJ1dG1DYW1wYWlnbiI6bnVsbCwidXRtQ29udGVudCI6bnVsbCwidXRtTWVkaXVtIjpudWxsLCJ1dG1Tb3VyY2UiOm51bGwsInV0bVRlcm0iOm51bGx9&utm_source=email&utm_medium=email&utm_campaign=fx-forgot-password&utm_content=fx-reset-password
```